data/json/obsolete.json

```diff
@@ -1639,5 +1639,21 @@
       [ "VISE", 1 ]
     ],
     "use_action": [ "HAMMER", "CROWBAR" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "knuckle_steel",
+    "symbol": "3",
+    "color": "dark_gray",
+    "name": { "str": "pair of steel knuckles", "str_pl": "pairs of steel knuckles" },
+    "description": "A mass of scrap metal crudely beat into shape, with folded rags underneath to protect the wearers knuckles.  A good, quick weapon - but you have to get within punching range to use it.",
+    "category": "weapons",
+    "material": [ "steel" ],
+    "volume": "250 ml",
+    "weight": "430 g",
+    "bashing": 4,
+    "price_postapoc": 50,
+    "qualities": [ [ "HAMMER", 1 ] ],
+    "flags": [ "UNARMED_WEAPON" ]
```

Comment:
You should add the recipe instead of the item here.

Reply:
Interesting. How does the game use it?

Reply:
Honestly I don't know, but old saves will show a debug message if the old recipe id is not obsoleted.